portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -330,20 +344,77 @@ public String getGatewaySiteId()
 	}
 
 	/**
+	 * {@inheritDoc}
+	 */
+	public String getSiteOrSectionTitle( Site site )
+	{
+		return getSiteOrSectionTitle( site, getProviderIDsForSite( site ) );
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public String getSiteOrSectionTitle( Site site, List<String> providerIDs )
+	{
+		String retVal = cms.determineSiteOrSectionTitle( site, providerIDs );
```

Comment:
Yep, sorry, it looks like all instances of Sakai now have CMS. My mental state hasn't quite got in sync with Sakai (https://jira.sakaiproject.org/browse/SAK-11685)